### PR TITLE
Update twilio2FA.py

### PR DESCRIPTION
**Pull request by syntrydy**
Fix some issue encountered during authentication.

### CODE REVIEW

File: Server/integrations/twilio_sms/twilio2FA.py [hunk 1]
```diff
@@ -104,10 +104,16 @@ def authenticate(self, configurationAttributes, requestParameters, step):
...
+                isVerified = foundUser.getAttribute("phoneNumberVerified")
+                if isVerified:
+                    mobile_number = foundUser.getAttribute("employeeNumber")
+                if  mobile_number == None:
+                    mobile_number = foundUser.getAttribute("mobile")
+                if  mobile_number == None:
+                    mobile_number = foundUser.getAttribute("telephoneNumber")
+                if  mobile_number == None: 
+                    print "TwilioSMS, Error finding mobile number for' % (user_name)
+
```
Comment (yurem) [inline Server/integrations/twilio_sms/twilio2FA.py]: After getting mobile_number you should restore lines:
```
                if isinstance(mobile_number, JSONArray):
                    mobile_number=mobile_number.get(0)
print "TwilioSMS, Message will be sent to number ", mobile_number
```

Comment (syntrydy) [reply Server/integrations/twilio_sms/twilio2FA.py]: i think this block is no more useful because the mobile_number is no more save in ldap as a list.

Comment (syntrydy) [reply Server/integrations/twilio_sms/twilio2FA.py]: Hence we don't need to test if it is an instance of a jsonArray.

Comment (syntrydy) [reply Server/integrations/twilio_sms/twilio2FA.py]: make sense?

**Pull request merged**